src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,13 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery extends ColorSupplier {
```

Comment:
Lottery shouldn't extend ColorSupplier.
We can create ColorSupplier object and use it
```
ColorSupplier colorSupplier = new ColorSupplier();
...
String color = colorSupplier.getRandomColor();
```